source/change-streams.rst

```diff
@@ -277,33 +306,48 @@ Driver API
      * This option is sent only if the caller explicitly provides a value. The
      * default is to not send a value.
      *
+     * @since 3.6
      * @see https://docs.mongodb.com/manual/reference/command/aggregate
      * @note this is an aggregation command option
      */
     collation: Optional<Document>;
 
+    /**
+     * The changeStream will only provides changes that occurred after the
+     * specified cluster timestamp. The default value is a cluster time obtained from
+     * the server before the changestream was created.
+     * @since 4.0
+     */
+    startAtClusterTime: Optional<Timestamp>
+
+    /**
+     * If true, will create a changeStream that operates on the entire cluster.
+     * Is used internally by MongoClient.watch.
```

Comment:
If it's for internal use only why add it to the public API of ChangeStreamOptions?  What would happen if the app sets this to "true" for either Collection.watch or Database.watch? I think it should be removed, and then refactor the spec to separate ChangeStreamOptions with the keys/values included in the $changeStream document.

Reply:
> refactor the spec to separate ChangeStreamOptions with the keys/values included in the $changeStream document.

I think it's worth having a section that discusses exactly which options are pertinent to `$changeStream`. That follows my intention with adding `@note this is an option of the `$changeStream` pipeline stage` in a previous PR, but takes it a step further.

Reply:
Good point. I will remove it

Reply:
comment removed